common/script/ops/clearPMs.js

```diff
@@ -1,6 +1,6 @@
 module.exports = function clearPMs (user) {
   user.inbox.messages = {};
-  user.markModified('inbox.messages');
+  // user.markModified('inbox.messages');
```

Comment:
Hmm it was still breaking, but I was doing some changes with the wrap function. Let me test it again.